### DISCUSSION TITLE
Problema di bidirezionalità

### DISCUSSION THREAD

**Discussion by doriangrey03** [localsend/localsend#1332] [L-Italiano]
Ciao a tutti, ho installato localsend su più host : 2 macchine windows 1 macchina linux 1 VM e 1 Android. Il problema è che la macchina linux vede tutti gli altri dispositivi e riesce a mandare file e messaggi, ma le altre non vedono la macchina linux, come mai?

**Comment by SebRoLENS** [comment]
Ciao ! Dovresti mettere un'eccezione al firewall per permettere al LocalSend sulla macchina linux di essere visto dagli altri dispositivi. Dopo questo procedimento in effetti diventa visibile, ma nel mio specifico caso trovo problemi nel trasferimento file, fammi sapere se a te invece funziona. 

Per permettere l'accesso alla porta:

installa ufw, come farlo dipende dalla tua distribuzione.

Permetti la specifica porta:

`sudo ufw allow 53317/udp`

`sudo ufw allow 53317/tcp`

Infine 

`sudo ufw enable`

Per controllare che tutto sia andato bene:

`sudo ufw status`

A questo punto, tutto dovrebbe funzionare correttamente.

**Reply by doriangrey03** [reply]
perfetto ti ringrazio, è tutto funzionante ora, anche il trasferimento files và a buon fine
